monifu/shared/src/main/scala/monifu/reactive/internals/operators/debounce.scala

```diff
@@ -45,7 +45,7 @@ private[reactive] object debounce {
         private[this] var hasValue = false
 
         locally {
-          scheduleNext(timeoutMillis)
+          scheduleNext(initialDelay.toMillis)
```

Comment:
Also note that my initial implementation of `debouce` was repeating elements. The current `debounce` on the other hand follows the behavior of Rx and will only emit a single element after silence. Therefore an `initialDelay` doesn't make sense on `debounce`. Unless we expose the `debounceRepeated` again.